### ISSUE TITLE
Jiffy bump has broken CouchDB on CentOS 6

### ISSUE THREAD

**Issue by wohali** [apache/couchdb#2517]
@davisp see https://ci-couchdb.apache.org/blue/organizations/jenkins/jenkins-cm1%2FFullPlatformMatrix/detail/master/70/pipeline/147

```
[2020-02-04T15:49:19.764Z] Compiling c_src/decoder.c
[2020-02-04T15:49:19.764Z] cc1: error: unrecognized command line option "-flto"
[2020-02-04T15:49:19.764Z] cc1: error: unrecognized command line option "-flto"
```

I don't think we can get a new enough gcc on that image to compile this :(

Also, there's no Python 2.7 on that image - only Python 3 - can you update `build-aux/show-test-results.py` to use Python 3 instead please? Thanks.

**Comment by davisp** [comment]
I just had a PR to fix this today:

https://github.com/davisp/jiffy/pull/185

Thought it appears that there's issues with the Erlang versions on travis breaking Travis so will have to check into that first.

**Comment by wohali** [comment]
Intended to be fixed by #2523

**Comment by wohali** [comment]
Closed by #2517
